website/html/views.py

```diff
@@ -22,6 +22,8 @@
 import shutil
 import subprocess
 import xmlrpclib
+import re

```

Comment:
This import slipped in from a previous commit (wrong fork, wrong branch, had to cherry pick, resolve conflicts, ...).
Anyways, I actually think it already slipped into the original [commit](https://github.com/kaushik1091/clearinghouse/pull/1/commits/74fad83406fa8a06a3384ba0f52622fc4434eb32) by accident. Because I can't see the `re` module being used.